agent/config/types.go

```diff
@@ -198,4 +198,10 @@ type Config struct {
 	// CgroupPath is the path expected by the agent, defaults to
 	// '/sys/fs/cgroup'
 	CgroupPath string
+
+	// RpsLimit specifies the steady state throttle for the task metadata endpoint
+	RpsLimit int
```

Comment:
Similar to above.  Can you please rename this to something like `TaskMetadataSteadyStateRate`?